config/environments/test.rb

```diff
@@ -28,6 +28,9 @@
   # Print deprecation notices to the stderr.
   config.active_support.deprecation = :stderr
 
+  # Disable cache in test
+  config.cache_store = :null_store
```

Comment:
Perhaps it's better to enable caching in test & development. There's a risk that there are cache-invalidation bugs that would only surface on production otherwise.